src/OpenTelemetry.AutoInstrumentation/.publicApi/net462/PublicAPI.Shipped.txt

```diff
@@ -10,8 +10,6 @@ OpenTelemetry.AutoInstrumentation.CallTarget.CallTargetState
 OpenTelemetry.AutoInstrumentation.CallTarget.CallTargetState.Activity.get -> System.Diagnostics.Activity?
 OpenTelemetry.AutoInstrumentation.CallTarget.CallTargetState.CallTargetState() -> void
 OpenTelemetry.AutoInstrumentation.CallTarget.CallTargetState.CallTargetState(System.Diagnostics.Activity! activity) -> void
-OpenTelemetry.AutoInstrumentation.CallTarget.CallTargetState.CallTargetState(System.Diagnostics.Activity! activity, object! state, System.DateTimeOffset? startTime) -> void
-OpenTelemetry.AutoInstrumentation.CallTarget.CallTargetState.CallTargetState(System.Diagnostics.Activity? activity, object! state) -> void
```

Comment:
Technically, you should keep it here, and mark it as REMOVED in Unshipped file.
Same for the second file.

Reply:
fixed in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3055/commits/e7ccb7aed42f565d54450939d9028c59aedc0e1f